native-sql-engine/cpp/compile.sh

```diff
@@ -8,6 +8,7 @@ STATIC_ARROW=${3:-OFF}
 BUILD_PROTOBUF=${4:-ON}
 ARROW_ROOT=${5:-/usr/local}
 ARROW_BFS_INSTALL_DIR=${6}
+BUILD_JEMALLOC=${7:-ON}
```

Comment:
should also define the option here?
https://github.com/oap-project/gazelle_plugin/blob/master/native-sql-engine/cpp/src/CMakeLists.txt#L22-L28

Reply:
Yes, you are right, I miss the default setting for BUILD_JEMALLOC.
Done to send a new commit to fix it.